default_payment_config.xml

```diff
@@ -29,13 +29,14 @@
         <wdoxidee_httppass>qD2wzQ_hrc!8</wdoxidee_httppass>
         <wdoxidee_descriptor>1</wdoxidee_descriptor>
         <wdoxidee_additional_info>1</wdoxidee_additional_info>
+        <initial_title>wd_heading_title_creditcard</initial_title>
     </payment>
     <payment>
       <oxid>wdpayolution-b2b</oxid>
       <oxactive>0</oxactive>
       <oxfromamount>10</oxfromamount>
       <oxtoamount>3500</oxtoamount>
-      <oxdesc id="wd_heading_title_payolution_b2b"/>
+      <oxdesc id="wd_heading_title_payolution_b2b_custom"/>
```

Comment:
Why is at the end now the custom? also i see the translation of titles doubled up?

Reply:
We added 3 new keys on Phraseapp (```heading_title_ratepayinvoice_custom```, ```heading_title_payolution_b2b_custom```, ```heading_title_payolution_invoice_custom```). These keys contain translations for payment method titles without "guaranted/garantiert" wording (it was a requirement from [Jira ticket](https://jira.parkside.at/browse/WDPD-216)). Those keys are used for consumer interface.

Reply:
hm.. can they be named is there a limit of the key length, i prefer it being named heading_title_ratepayinvoice_customer ?

Reply:
Done 👍